packages/adapter-mariadb/src/mariadb.ts

```diff
@@ -149,6 +149,10 @@ export class PrismaMariaDbAdapter extends MariaDbQueryable<mariadb.Pool> impleme
   async dispose(): Promise<void> {
     await this.client.end()
   }
+
+  underlyingDriver(): mariadb.Pool {
```

Comment:
I'm wondering if we really want to use a generic name consistent between adapters or if it would actually be better to specialize them based on the type name that the method returns (i.e. `getPool`, `getClient` etc).